modules/standard/AutoMath.chpl

```diff
@@ -238,137 +238,287 @@ module AutoMath {
   }
 
 
```

Comment:
Should the documentation for these functions be removed? Since the deprecation doesn't link directly to the function, just to the module, it may make navigating/understanding the documentation harder.

Reply:
That's a good point.  I had been removing the documentation in the interest of minimizing the clutter from so many deprecated symbols still being present so that the maintained symbols would be easier to find, but you're right that the documentation does provide value.  I'll restore it